calendar/src/Components/EventForm/EventForm.tsx

```diff
@@ -0,0 +1,182 @@
+import { useReducer, useRef, useState } from "react";
+import { FormData } from "../../Utils/database";
+import "./event.css";
+
+import timeIcon from "../../images/schedule_FILL0_wght400_GRAD0_opsz48.svg";
+import noteIcon from "../../images/notes_FILL0_wght400_GRAD0_opsz48.svg";
+import Input, { InputTypes } from "./Input";
+import InputValidator from "./InputValidator";
+import FormBlock from "./FormBlock";
+
+const TIME_VALIDATION_ERROR_MESSAGE = "Event cannot end before it starts.";
+const TITLE_VALIDATION_ERROR_MESSAGE = "Please enter a title.";
+
+const convertInputToDate = (dateString: string, timeString: string): number => {
+	return Date.parse(`${dateString} ${timeString}`);
+};
+
+export interface EventFormProps {
+	hideModal: Function;
```

Comment:
The type should probably be `() => void`. Writing `Function` is basically saying any type of function and we would like to have more strict types.
If possible fix in all usages where you write type `Function`.